pkgdb2client/admin.py

```diff
@@ -461,6 +497,8 @@ def do_process(args):
             data = __handle_request_package(actionid, action)
         elif action['action'] == 'request.branch':
             data = __handle_request_branch(actionid, action)
+        elif action['action'] == 'request.unretire':
+            data = __handle_request_unretire(actionid, action)
         else:
             print('Action %s not supported by pkgdb-cli' % action['action'])
```

Comment:
I'm realizing that the message is wrong here, since it's ``pkgdb-admin`` not ``pkgdb-cli`` :)